staff/lgimeneza/app-template/home.html

```diff
@@ -1,40 +0,0 @@
-<!DOCTYPE html>
```

Comment:
it seems that you have deleted this challenge by accident! please, push these files again back into your repo (they will be added back to the pr automatically).

Reply:
+1